javascript/modules/client/src/client.ts

```diff
@@ -282,32 +308,124 @@ export class Client extends ClientCore implements IClient {
       override.value = amount;
     }
 
-    return daoInstance.estimateGas.deposit(
-      tokenAddress,
-      amount,
-      reference,
-      override,
-    ).then((gasLimit) => {
-      return this.web3.getApproximateGasFee(gasLimit.toBigInt());
-    });
+    return daoInstance.estimateGas
+      .deposit(tokenAddress, amount, reference, override)
+      .then(gasLimit => {
+        return this.web3.getApproximateGasFee(gasLimit.toBigInt());
+      });
+  }
+
+  private async _getDaoBalances(
+    daoIdentifier: string
```

Comment:
```suggestion
  private async _getBalances(
    daoIdentifier: string
```

Reply:
On top of that, I also suggest receiving an optional list of token addresses, to be used in addition to the well knows ones.

Reply:
Also, async not needed